### PR TITLE
Add Web-Planner to docs/references/planners

### PR DESCRIPTION
**Pull request by Maumagnaguagno**
This is my attempt to include Web-Planner to the wiki, as discussed in #77.

I was going for a simple inclusion in the additional resources, but a planner template was recommended.
The template includes a Jekyll front matter, in which I used lower case ``title`` to match the permalink and a ``nav_order`` of 100, as this seems to be used to sort planners and Web-Planner should be the last in alphabetical order.
I am not sure about these two elements, feel free to modify them to match your style.

### CODE REVIEW

File: docs/reference/planners/Web-Planner/main.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Page Contributors: {% git_author %}
+
+{% planner_tag Optimising Satisfycing %}
+
+Year Published: 2017
+
+Paper: [WEB PLANNER: A Tool to Develop Classical Planning Domains and Visualize Heuristic State-Space Search](http://icaps17.icaps-conference.org/workshops/UISP/uisp17proceedings.pdf#page=36) [Magnaguagno, M. Pereira, R. M�re, M. and Meneguzzi, F.]
+
+Similar to [Planning.Domains](#planning.domains), it is a planner in the cloud with editor, verification, validation and visualization capabilities.
```
Comment (meneguzzi) [inline docs/reference/planners/Web-Planner/main.md:18]: Suggestion to change:

```Similar to [Planning.Domains](#planning.domains), it is a planner in the cloud with editing, verification, validation and visualization capabilities.```

File: docs/reference/planners/Web-Planner/main.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- [Knowledge Engineering Tools and Techniques for AI Planning](https://www.springer.com/gp/book/9783030385606) - chapter 11, Springer 2020
+
+## Support
+
+The planner supports ``:strips``, ``:negative-preconditions``, ``:equality`` and ``:typing`` requirements.
+Types are not required for fast/compact grounding, rigid preconditions for every parameter are enough.
+
+## Additional Notes
+
+Visualization capabilities of Web-Planner are also available as plugins for Planning.Domains:
```
Comment (meneguzzi) [inline docs/reference/planners/Web-Planner/main.md:33]: Suggestion for change:
```The visualization capabilities of Web-Planner are also available as plugins for Planning.Domains:```

**Review by meneguzzi — COMMENTED** [review]
Here are my two cents, for what it's worth.

**Review by nergmada — CHANGES_REQUESTED** [review]
The planning wiki is designed to be a centralised source of planning material. As such, there should be less "look at this source for more information" and more "here's roughly what the web planner does under the hood in layman's terms, here are relevant citations".

File: docs/reference/planners/Web-Planner/main.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+{% planner_tag Optimising Satisfycing %}
+
+Year Published: 2017
+
+Paper: [WEB PLANNER: A Tool to Develop Classical Planning Domains and Visualize Heuristic State-Space Search](http://icaps17.icaps-conference.org/workshops/UISP/uisp17proceedings.pdf#page=36) [Magnaguagno, M. Pereira, R. M�re, M. and Meneguzzi, F.]
+
+Similar to [Planning.Domains](#planning.domains), it is a planner in the cloud with editor, verification, validation and visualization capabilities.
+The planner is available at https://web-planner.herokuapp.com/
+For more information see:
+- [DOVETAIL - An Abstraction for Classical Planning Using a Visual Metaphor](https://www.aaai.org/ocs/index.php/FLAIRS/FLAIRS16/paper/viewFile/12966/12544) - FLAIRS 2016
```
Comment (nergmada) [inline docs/reference/planners/Web-Planner/main.md:21]: A relevant paragraph should be included rather than a link to other material

File: docs/reference/planners/Web-Planner/main.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- [WEB PLANNER: A Tool to Develop Classical Planning Domains and Visualize Heuristic State-Space Search](http://icaps17.icaps-conference.org/workshops/UISP/uisp17proceedings.pdf#page=36) - ICAPS/UISP 2017
+- [Develop, Visualize and Test Classical Planning Descriptions in your Browser](http://www.meneguzzi.eu/felipe/pubs/icaps-webplan-demo-2019.pdf) - ICAPS/Demos 2019.
+- [Knowledge Engineering Tools and Techniques for AI Planning](https://www.springer.com/gp/book/9783030385606) - chapter 11, Springer 2020
+
+## Support
+
+The planner supports ``:strips``, ``:negative-preconditions``, ``:equality`` and ``:typing`` requirements.
```
Comment (nergmada) [inline docs/reference/planners/Web-Planner/main.md:28]: Support should use the full table, with yes no values in order to maintain standards across the wiki. If there are pertinent facts to support (such as certain limitations) these can be included as a paragraph at the bottom or top of the ## Support section

**Review by nergmada — CHANGES_REQUESTED** [review]

File: docs/reference/planners/Web-Planner/main.md (hunk 4 above)
Comment (Maumagnaguagno) [reply docs/reference/planners/Web-Planner/main.md]: I added a support table based on the one from FD.

File: docs/reference/planners/Web-Planner/main.md (hunk 3 above)
Comment (Maumagnaguagno) [reply docs/reference/planners/Web-Planner/main.md:30]: I added more content to describe the system itself, not sure if I should add a paragraph for each related publication.

**Review by nergmada — APPROVED** [review]
